tests/target/issue-4210.rs

```diff
@@ -0,0 +1,15 @@
+// rustfmt-wrap_comments: true
+
+/// Table that is > 80 symbols:
+///
+/// | table | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+/// |-------|-----------------------------------------------------------------------------|
+/// | val   | x                                                                           |
+pub struct Item;
+
+/// Table that is > 80 symbols:
+///
+/// | table | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+/// |-------|-----------------------------------------------------------------------------
+/// | val   | x                                                                          
+pub struct Item;
```

Comment:
I think these are good tests ported over from the backport, however I'd like to build on these before merging.

I'd like to see a test case for `wrap_comments=false` (just for completeness). Additionally, I'd like to add some text before the table that gets wrapped when `wrap_comments=true` to further emphasize that the table isn't being wrapped even when the text is wrapped.

I also think we should add a row to the table (besides the header) that contains a long line to show that it also isn't wrapped.